libpam/pam_auth.c

```diff
@@ -36,7 +36,7 @@ int pam_authenticate(pam_handle_t *pamh, int flags)
     if (retval != PAM_INCOMPLETE) {
 	_pam_sanitize(pamh);
 	_pam_await_timer(pamh, retval);   /* if unsuccessful then wait now */
-	D(("exit"));
+	D(("exiting"));
```

Comment:
If adding a period for `called.` consider adding it for `exited.` too; or remove for both.

Looking at things, probably dropping the period on entry looks best IMO.

Reply:
Looking at majority of files, "called." and "exiting" are preferred by now.